src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
+const MAX_HEALTH = 100;
+const DAMAGE = 50;
+
 class Animal {
-  // write your code here
+  static alive = [];
+  health = MAX_HEALTH;
+
+  constructor(name) {
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
```

Comment:
1) u will have the same initial value in each instance.
2) In this case, we may not call the constructor
```suggestion
  hidden = false
```